source/usage-examples/findOne.txt

```diff
@@ -3,3 +3,61 @@ Find a Document
 ===============
 
 .. default-domain:: mongodb
+
+You can retrieve a single document from your collection with the
+``collection.findOne()`` method. 
+
+The ``findOne()`` method takes a **context**, a **filter**, and a
+pointer to a configuration object as arguments.
+
+A context is a value of type ``context.Context``. Contexts are useful in
+concurrent server programming. A discussion of the context pattern and the
+``context`` package is outside the scope of this page, but if you are interested
+you can find more information
+`in this article on The Go Blog <https://blog.golang.org/context>`__.
```

Comment:
I think it might be worth making a page for this, as a large portion of the go driver codebase seems to interact with ``context.Context`` values. I have [made a ticket](https://jira.mongodb.org/browse/DOCSP-15527) for the Context page, added a place holder for the context page in the fundamentals section, and moved the section such that it is described last in the description of the `findOne()` method parameters. Fix: https://github.com/mongodb/docs-golang/pull/2/commits/70cacf45b53ca05072ac20cec62b90df79be7642